### DISCUSSION TITLE
nITF nitro pixel lat lon question

### DISCUSSION THREAD

**Discussion by sherrick01821** [mdaus/nitro#386] [Q&A]
The nitro pixel lat lon sub header for SNIP compliance requires to have 2 bands but the pixel lat lon are supposed to be packed together. If I have 2 bands then I need to provide 2 image sources. How do I do that? Is one band lat and the other band lon? I'm confused about the packing together of lat and lon?

**Comment by bradh** [comment]
Assuming you are talking about PIX_LATLON, I'd suggest building it by hand (i.e. with custom code). Creating an image source of latitudes and an image source of longitudes doesn't seem like the best way to approach the problem.

**Comment by sherrick01821** [comment]
Brad
Thanks for the reply. So by telling nitro you have 2 bands requires 2 band sources and it looks like the lat lons need to be packed together ( ( lat,lon), ( lat,lon) , etc.. )  What should iimage source 1 have in it and what should image source 2 have in it?
Thanks,
Steve

    On Thursday, July 29, 2021, 7:26:45 PM EDT, Brad Hards ***@***.***> wrote:  
 
 


Assuming you are talking about PIX_LATLON, I'd suggest building it by hand (i.e. with custom code). Creating an image source of latitudes and an image source of longitudes doesn't seem like the best way to approach the problem.

**Reply by bradh** [reply]
I guess that I didn't communicate it very well (since you've asked the same question again), and I'm not sure how else to express it, so sorry I couldn't help.

**Comment by sherrick01821** [comment]
I'm going to edit my nitro code to not write out both bands and then look at the file with ENVI and see what it says

    On Friday, July 30, 2021, 8:23:02 PM EDT, Brad Hards ***@***.***> wrote:  
 
 


I guess that I didn't communicate it very well (since you've asked the same question again), and I'm not sure how else to express it, so sorry I couldn't help.
